packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -127,6 +127,12 @@ export class DefaultPageBlockComponent
     .affine-default-page-block-title-container {
       margin-top: 78px;
     }
+
+    // This css is used to make text in title line break automatically,
+    // but in Virgo there is still just one line so the link break from this css is just visual.
+    .affine-default-page-block-title virgo-unit-text > span {
+      white-space: pre-line !important;
```

Comment:
And I think you can remove the `!important` after fix CSS comment

Reply:
There are inline styles in `span` of `virgo-unit-text` so I use `!important` cover it.